torch/csrc/jit/import.cpp

```diff
@@ -258,15 +258,25 @@ at::Tensor ScriptModuleDeserializer::loadTensor(
   }
 
   at::Tensor result;
-  if (device.type() == at::DeviceType::CPU) {
-    result =
-        at::empty({0}, at::CPU(type).options())
-            .set_(storage_it->second, tensor_proto.offset(), dims, strides);
-  } else if (device.type() == at::DeviceType::CUDA) {
-    result =
-        at::empty(
-            {0}, c10::TensorOptions(type).device(storage_it->second.device()))
-            .set_(storage_it->second, tensor_proto.offset(), dims, strides);
+  if (tensor_proto.is_quantized()) {
```

Comment:
@jerryzh - do we support on-device quantized tensors? if so  - we should merge it with the if below

Reply:
what is on-device quantized tensor?

Reply:
Sorry, I meant e.g. CUDA quantized tensor. But given that this code is likely going to change in the future with serialization overhaul, I guess it's fine to have it as is.